commit-event-service/src/test/scala/io/renku/commiteventservice/events/categories/globalcommitsync/eventgeneration/LatestCommitFinderSpec.scala

```diff
@@ -48,40 +49,27 @@ class LatestCommitFinderSpec
 
   "findLatestCommitId" should {
     "return the latest CommitID if remote responds with OK and valid body - personal access token case" in new TestCase {
-      val personalAccessToken = personalAccessTokens.generateOne
+      implicit val personalAccessToken = personalAccessTokens.generateSome
```

Comment:
More or less as in the other finders, we could maybe generate an accessToken instead of a concrete personalAccessToken